### PR TITLE
Let's see if this fix solves our issue

### PR DESCRIPTION
**Pull request by pranjalkaler**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
-echo "=============================="
+echo "======================================================="
+echo "                 Build Details for Java                "
+echo "                   Compiled $count_java files            "
+echo "             $errorCountJava files failed during build        "
+echo "======================================================="
+
+if [[ $errorCountCpp -eq 0 ]] && [[ $errorCountJava -eq 0 ]] ; then
+    echo "BUILD SUCCESS"
+    exit 1
```
Comment (pranjalkaler) [inline build.sh:46]: this needs to be in else. But at least it gives failure or success output

Comment (pranjalkaler) [reply build.sh:46]: updating this

**Comment by pranjalkaler** [comment]
Accidently pushed exit 1 for success check. Build failed. Might solve our purpose temporarily

File: build.sh (hunk 1 above)
Comment (pranjalkaler) [reply build.sh]: https://github.com/vishalv971/CompetitiveCoding/runs/1052235126?check_suite_focus=true

**Comment by SaumyaSachdev** [comment]
Looks good to me!

**Review by SaumyaSachdev — APPROVED** [review]

File: build.sh [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
 
-for fn in `find . -name "*.cpp"`; do 
-	echo "Currently compiling $fn"
-	g++ $fn
+
+errorCountCpp=0
+errorCountJava=0
+for fn in `find . -name "*.cpp"`; do
+    g++ $fn
+    if [ $? -eq 0 ]; then
```
Comment (vishalv971) [inline build.sh:8]: I was thinking to use $? itself , nice job @pranjalkaler

**Review by vishalv971 — APPROVED** [review]
Looks good, i merge now.

**Pull request merged**